doc/developer.md

```diff
@@ -45,40 +48,50 @@ We recommend this for development work ONLY.
     openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/httpd/ssl/apache.key -out /etc/httpd/ssl/apache.crt
     ```
 
-7. Copy the configuration files

```

Comment:
These sections are not required as now symlinking, right?

Reply:
the parts of the old Section 7 that relate to the REST interface have moved to the new Section 8 where they are symlinked.

the other parts of the old Section 7 and all the old Section 8 refer to files needed for an APEL server, so don't need to be copied.